client/components/Alert/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from 'styled-components';
+import tw from 'tailwind.macro';
+
+const Container = styled.div`
+  ${tw`p-4 rounded-sm border`}
+`;
+
+const ErrorAlert = styled(Container)`
+  ${tw`border-red bg-pink-200`}
+`;
+
+const WarningAlert = styled(Container)`
+  ${tw`border-yellow bg-yellow-200`}
+`;
+
+const SuccessAlert = styled(Container)`
+  ${tw`border-green bg-green-200`}
+`;
+
+const InfoAlert = styled(Container)`
+  ${tw`border-blue bg-blue-300`}
+`;
```

Comment:
Make sure the style can make the component looks like this

![image](https://user-images.githubusercontent.com/12707960/74125862-01fd1000-4c09-11ea-82f7-28bb9bc34d27.png)